### ISSUE TITLE
Focus appearance issue with sub-components

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag#2602]
The line just before the exceptions in 2.4.11 Focus Appearance reads:

Where a user interface component has active sub-components, if a sub-component receives a focus indicator, these requirements may be applied to the sub-component instead.

As written this seems to provide an exception for sub-components. Are there situations where the focused sub-components don't need to show the focus? The first note seems to suggest this but I can't think of a situation where this comes up and I think that the tablist example in the understanding document only shows a tablist with unfocusable subcomponents. It seems that if subcomponents are focusable, this needs to apply, so I'm suggesting:

Where a user interface component has active sub-components, if a sub-component receives a focus indicator, these requirements apply to the sub-component.

**Comment by yatil** [comment]
@awkawk I wondered about this [in my live stream yesterday](https://www.youtube.com/watch?v=d_qYCV0WSRs) and @alastc explained that the sub-components are “selected”, not focused. So in that case the sub-component would not receive a focus indicator but a selection indicator. (And apparently that is something entirely different.)

![YouTube chat screenshot](https://user-images.githubusercontent.com/58862/184355222-9aa0ac64-cb5e-4cb3-8884-32cdf87536d1.png)

<details><summary>Screenshot Alternative Text</summary>

**29:08 Alastair Campbell** Eric - some widgets would use a 'selection' rather than focus, which is ok.

**30:09 Alastair Campbell** E.g. a select box has items that you are selecting, and the overall box would have focus

</details>

**Comment by alastc** [comment]
There's a section in the understanding doc about this: https://w3c.github.io/wcag/understanding/focus-appearance.html#keyboard-focus

There are several instances where the sub-component (e.g. select, tab) is highlighting the "selected" state, not the focused state. (Coming under non-text contrast.)

There are also large grids (e.g. tables) where the item in focus in the table, and you arrow around the cells (sub-components). Requiring a massive indicator around the table did not seem a good idea. It might be useful, but does not seem to be something that should be required.

It is very difficult to be emphatic about which must have a focus indicator, so it seemed best to provide some flexibility and allow the author to decide which has the focus indicator.

**Comment by awkawk** [comment]
The selected item may be focused and selected, or just selected. When I have a list box I need to be able to indicate whether  list items (potentially multiple items in a multi-select list) are focused for keyboard navigation, and selection alone doesn't do that because the selection is persistent beyond the time of direct interaction.

**Comment by bruce-usab** [comment]
Thanks for raising this @awkawk since the concept of a <q>selection indicator</q> &ndash; @yatil term &ndash; is a bit odd. 
 From Editors Draft note (emphasis in original):

> However, it may also be possible to indicate user interaction for such sub-components by relying strictly on a visual indication of which item is _selected_.

Instead of <q>is selected</q> (which is a state) might this be better framed as an action?  For example:

> However, it may also be possible to indicate user interaction for such sub-components by relying strictly on a visual indication when an item is selected or activated.

**Comment by awkawk** [comment]
@alastc I checked with James Nurthen and he pointed me to this ARIA document on implementing keyboard support: https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#x6-3-focus-vs-selection-and-the-perception-of-dual-focus.

No focus, no selection:
![Screen Shot 2022-08-12 at 3 33 18 PM](https://user-images.githubusercontent.com/4228257/184430440-95d84717-3c15-42c9-b135-23af95e60fe7.png)

Only top-level of component focused:
![Screen Shot 2022-08-12 at 3 32 00 PM](https://user-images.githubusercontent.com/4228257/184430244-441f1967-dc93-4c1e-a1de-f6e6dead957b.png)

Focus on the first option in the list, which is also active and is selected:
![Screen Shot 2022-08-12 at 3 32 06 PM](https://user-images.githubusercontent.com/4228257/184430240-37c3a211-46a7-41dc-9127-972a56e0383e.png)

Focus is outside of the listbox, but the first item is not active and is selected:
![Screen Shot 2022-08-12 at 3 32 12 PM](https://user-images.githubusercontent.com/4228257/184430235-284932b4-d2aa-4b18-b396-663377a147de.png)

Hamster option is focused but not selected. Dog option is selected/active but not focused:
![Screen Shot 2022-08-12 at 3 31 23 PM](https://user-images.githubusercontent.com/4228257/184430246-de5733c1-a4a3-4502-9b37-58eacc4885e0.png)

**Comment by Nidhi2003** [comment]
assign

**Comment by alastc** [comment]
The PR for this was merged, so closing.
